src/backend/distributed/sql/udfs/citus_finish_pg_upgrade/11.1-1.sql

```diff
@@ -104,7 +104,7 @@ BEGIN
     --
     -- register triggers
     --
-    FOR table_name IN SELECT logicalrelid FROM pg_catalog.pg_dist_partition
+    FOR table_name IN SELECT logicalrelid FROM pg_catalog.pg_dist_partition JOIN pg_class ON (logicalrelid = oid) WHERE relkind <> 'f'
```

Comment:
The only difference between `11.1-1.sql` and 11.0-4.sql` is one line

    PERFORM columnar_internal.columnar_ensure_am_depends_catalog();
vs

    PERFORM citus_internal.columnar_ensure_am_depends_catalog();

Is this really expected?

Reply:
This file existed before this PR.
The difference you mentioned was made on #5911